Framework/Algorithms/src/Q1DWeighted.cpp

```diff
@@ -49,7 +49,7 @@ void Q1DWeighted::init() {
   auto monoValidator = std::make_shared<CompositeValidator>(CompositeRelation::AND);
   auto tofValidator = std::make_shared<CompositeValidator>(CompositeRelation::AND);
 
-  monoValidator->add<WorkspaceUnitValidator>("Empty");
+  monoValidator->add<WorkspaceUnitValidator>("Label");
```

Comment:
Could you add a comment and perhaps a note about this change in the validation? Is it actually relevant for the execution what unit is there? Could we have both `Empty` and `Label`?

Reply:
No, it is not actually relevant, we just wanted to put a barrier so that only relevant files could be processed by Q1DWeighted. But empty is as fine as label or anything, in truth (actually Wavelength is not fine because we rely on that to know if we are processing monochromatic data because ... reasons i guess)